.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+vendor
```

Comment:
alternative: not have it in gitignore at all. example, if while de-bugging I make a change to a file in vendor, I want git to track that